test/test_grid.py

```diff
@@ -336,6 +350,26 @@ def test_calculate_total_face_area_file(self):
 
         nt.assert_almost_equal(area, constants.MESH30_AREA, decimal=3)
 
+    def test_calculate_total_face_area_sphere(self):
+        """Computes the total face area of an MPAS mesh that lies on a unit
+        sphere, with an expected total face area of 4pi."""
+        mpas_grid_path = current_path / 'meshfiles' / "mpas" / "QU" / 'mesh.QU.1920km.151026.nc'
+
+        ds = xr.open_dataset(mpas_grid_path)
+        primal_grid = ux.Grid(ds, use_dual=False)
+        dual_grid = ux.Grid(ds, use_dual=True)
+
+        primal_face_area = primal_grid.calculate_total_face_area()
+        dual_face_area = dual_grid.calculate_total_face_area()
+
+        nt.assert_almost_equal(primal_face_area,
+                               constants.UNIT_SPHERE_AREA,
+                               decimal=3)
+
+        nt.assert_almost_equal(dual_face_area,
+                               constants.UNIT_SPHERE_AREA,
+                               decimal=3)
+
```

Comment:
@areanddee

The behavior is now expected for MPAS grids that lie on a unit sphere.